build/bakefiles/config.bkl

```diff
@@ -132,6 +132,14 @@ Accepted values: IA64, X64, ARM64
 (AMD64 accepted as synonym for X64 but should not be used any more).
             </description>
         </option>
+        <option name="VS_VER">
+            <!-- defined by VS command prompt, can (at least) be 14.0, 15.0, 16.0, ... -->
```

Comment:
I initially misunderstood this as meaning that `VS_VER` itself was supposed to be defined in the environment. Now I see that it's not the case and that its value just comes from `VisualStudioVersion` environment variable, but maybe it would be better to reword this to avoid the confusion?

Also, while checking this I noticed that MSVS 2017 and 2019 define `VSCMD_ARG_HOST_ARCH` and `VSCMD_ARG_TGT_ARCH` env vars too (and `VSCMD_VER`, but this is less important). Unfortunately MSVS 2015 doesn't have those.

Reply:
I renamed the options.

I saw those env vars too, and indeed 2015 does not have them.